### PR TITLE
Write documentation (by po)

### PR DESCRIPTION
**Pull request by MarcoKlerks**
Writing documentation (Read the Docs)

(Significant) updates to:
- [x] Introduction
- [ ] API-specifications
- [x] Admin interface
- [x] > Publicaties
- [x]  > Metadata
- [ ]  > Accounts
- [ ]  > Logging
- [ ]  > Configuratie
- [ ]  > Handige links
- [ ]  > Overige
- [ ] Installation
- [ ] Developers
- [ ] Versioning
- [ ] Release notes

### CODE REVIEW

**Comment by MarcoKlerks** [comment]
@sergei-maertens Ik heb wat teksten geschreven voor de handleiding. Voordat ik verder ga, ben ik benieuwd naar je feedback. Dat RST-gebeuren is nieuw voor mij.

Eén check faalt. Ik heb géén idee waarom. :-)

File: docs/admin/Accounts/index.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.. admin_Accounts_index:
+
+Accounts
+=========
```
Comment (sergei-maertens) [inline docs/admin/Accounts/index.rst:4]: One "finicky" thing about RST is that the amount of `=` characters should be the same as the length of the title:

```suggestion
Accounts
========
```

File: docs/admin/Configuratie/index.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.. admin_Configuratie_index:
+
+Configuratie
+=============
```
Comment (sergei-maertens) [inline docs/admin/Configuratie/index.rst:4]:
```suggestion
Configuratie
============
```

File: docs/admin/Handige_links/index.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.. admin_Handige_links_index:
+
+Handige links
+==============
```
Comment (sergei-maertens) [inline docs/admin/Handige_links/index.rst:4]:
```suggestion
Handige links
=============
```

File: docs/admin/Logging/index.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.. admin_Logging_index:
+
+Logging
+========
```
Comment (sergei-maertens) [inline docs/admin/Logging/index.rst:4]:
```suggestion
Logging
=======
```

File: docs/admin/Metadata/index.rst [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+.. _`audittrail` : ../Logging/index.hmtl
+.. _`Wet open overheid` : https://wetten.overheid.nl/BWBR0045754/
+.. _`Woo-index`: https://open.overheid.nl/
+.. _`DiWoo` : https://standaarden.overheid.nl/diwoo/metadata
+.. _`Woo informatiecategorieën` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
+.. _`Thema-indeling voor Officiële Publicaties (TOP-lijst)` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
+.. _`audittrail` : ../Logging/index.hmtl
+
+Metadata
+============
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:12]:
```suggestion
Metadata
========
```

File: docs/admin/Metadata/index.rst [hunk 6]
```diff
@@ -0,0 +1,115 @@
+.. admin_Metadata_index:
+
+.. _`audittrail` : ../Logging/index.hmtl
+.. _`Wet open overheid` : https://wetten.overheid.nl/BWBR0045754/
+.. _`Woo-index`: https://open.overheid.nl/
+.. _`DiWoo` : https://standaarden.overheid.nl/diwoo/metadata
+.. _`Woo informatiecategorieën` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
+.. _`Thema-indeling voor Officiële Publicaties (TOP-lijst)` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
+.. _`audittrail` : ../Logging/index.hmtl
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:9]: we tend to put the link targets at the bottom of a file usually to increase readability of the code

additionally, you don't need to wrap the aliases in backticks for the link definitions :)

```suggestion
.. _Wet open overheid: https://wetten.overheid.nl/BWBR0045754/
.. _Woo-index: https://open.overheid.nl/
.. _DiWoo: https://standaarden.overheid.nl/diwoo/metadata
.. _Woo informatiecategorieën: https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
.. _Thema-indeling voor Officiële Publicaties (TOP-lijst): https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
```

This also removes the internal links, for those we can use the `ref` syntax, e.g.:

```rst
:ref:`admin_logging_index`
```

File: docs/admin/Metadata/index.rst [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+Metadata
+============
+
+Onder het menu-item "Metadata" en op het dashboard onder het kopje "Metadata" wordt toegang geboden tot het beheer van:
+
+* Informatiecategorieën
+* Thema's
+
+Door herop te klikken wordt het desbetreffende beheerscherm geopend.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:19]:
```suggestion
Door hierop te klikken wordt het desbetreffende beheerscherm geopend.
```

File: docs/admin/Metadata/index.rst [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+Informatiecategorieën
+------------------------
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:22]:
```suggestion
Informatiecategorieën
---------------------
```

File: docs/admin/Metadata/index.rst [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+Toelichting
+~~~~~~~~~~~~~
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:25]:
```suggestion
Toelichting
~~~~~~~~~~~
```

File: docs/admin/Metadata/index.rst [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+De `Wet open overheid`_ beschrijft 17 *informatiecategorieën*. Documenten die onder minstens één van deze categorieën vallen, moeten *actief* openbaar gemaakt worden. Dat wil zeggen dat een bestuursorgaan deze online moet publiceren, ook wanneer daar niet door een burger expliciet om gevraagd is in een *Woo-verzoek*. 
+
+Daarnaast vraagt de wet om ook documenten openbaar te maken die buiten deze *informatiecategorieën* vallen. Hiervoor geldt geen "harde" plicht, maar wel een *inspanningsverplichting*.
+
+Overigens gelden op de plicht tot openbaarmaking ook een aantal uitzonderingen (hoofdstuk 5 van de wet). Raadpleeg voor meer informatie over de wet de jurist en/of informatiebeheerder van uw eigen organisatie.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:31]:
```suggestion
Overigens gelden op de plicht tot openbaarmaking ook een aantal uitzonderingen (hoofdstuk 5 van de wet). Raadpleeg voor meer informatie over de wet de jurist en/of informatiebeheerder van je eigen organisatie.
```
recent developments show that information language (in user interfaces, documentation...) is preferred

File: docs/admin/Metadata/index.rst [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+
+Voor de aansluiting op de landelijke `Woo-index`_ wordt ten zeerste geadviseerd om documenten te voorzien van metadata conform de `DiWoo`_-standaard. Deze standaard vereist omder meer dat in de metadata wordt vastgelegd op grond van welke *informatiecategorie(ën)* het document actief openbaar is gemaakt.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:33]:
```suggestion
Voor de aansluiting op de landelijke `Woo-index`_ wordt ten zeerste geadviseerd om documenten te voorzien van metadata conform de `DiWoo`_-standaard. Deze standaard vereist onder meer dat in de metadata wordt vastgelegd op grond van welke *informatiecategorie(ën)* het document actief openbaar is gemaakt.
```

File: docs/admin/Metadata/index.rst [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+
+De 17 *informatiecategorieën* zijn opgenomen in de landelijke waardelijst `Woo informatiecatgeorieën`_. Deze lijst is al standaard opgenomen in het "Woo Publications"-component. Daarnaast bestaat de mogelijkheid, zoals we hieronder zullen beschrijven, om in het kader van de *inspanningsverplichting* hier voor de eigen organisatie *extra informatiecategorieën* aan toe te voegen.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:35]:
```suggestion
De 17 *informatiecategorieën* zijn opgenomen in de landelijke waardelijst `Woo informatiecatgeorieën`_. Deze lijst is al standaard opgenomen in het "Woo Publicaties"-component. Daarnaast bestaat de mogelijkheid, zoals we hieronder zullen beschrijven, om in het kader van de *inspanningsverplichting* hier voor de eigen organisatie *extra informatiecategorieën* aan toe te voegen.
```

File: docs/admin/Metadata/index.rst [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+In het "WOO Publications"-component leggen we de *informatiecategorieën* vast op de *publicatie* (in ontwikkeling). Dezelfde *informatiecategorieën* gelden voor de daaraan gekoppelde *documenten*.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:37]:
```suggestion
In het "WOO Publicaties"-component leggen we de *informatiecategorieën* vast op de *publicatie* (in ontwikkeling). Dezelfde *informatiecategorieën* gelden voor de daaraan gekoppelde *documenten*.
```
we passen de NL vertalingen ook in de interface toe :)

File: docs/admin/Metadata/index.rst [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+
+Beheerscherm
+~~~~~~~~~~~~~~
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:40]:
```suggestion
Beheerscherm
~~~~~~~~~~~~
```

File: docs/admin/Metadata/index.rst [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+
+In het beheerscherm van de *informatiecatgeorieën* wordt een lijst getoond van alle *informatiecategorieën* die zijn opgeslagen in het "WOO Publications"-component.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:42]:
```suggestion
In het beheerscherm van de *informatiecatgeorieën* wordt een lijst getoond van alle *informatiecategorieën* die zijn opgeslagen in het "WOO Publicaties"-component. 
```

File: docs/admin/Metadata/index.rst [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+~~~~~~~~~~~~~~
+
+In het beheerscherm van de *informatiecatgeorieën* wordt een lijst getoond van alle *informatiecategorieën* die zijn opgeslagen in het "WOO Publications"-component. 
+
+Standaard staan de landelijke 17 verplichte *informatiecategorieën* hier al in. Zij zijn te herkennen aan de waarde "Waardelijst" in de kolom `Oorsprong` van de lijst. Zij kunnen niet gewijzigd worden.
+Daarnaast bestaat er de mogelijkheid om voor de eigen organisatie *extra informatiecategorieën* toe te voegen. Deze zijn te herkennen aan de waarde "Zelf-toegevoegd item" in de kolom `Oorsprong` van de lijst. Deze kunnen indien nodig gewijzigd worden.
+
+Op het beheerscherm zijn de volgende acties mogelijk:
+
+* Rechtboven zit een knop **"informatiecategorie toevoegen"** waarmee een *informatiecategorie* toegevoegd kan worden.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:49]:
```suggestion
* Rechtboven zit een knop **informatiecategorie toevoegen** waarmee een *informatiecategorie* toegevoegd kan worden.
```
quotes zijn niet nodig door de bold formatting, dat maakt de documentatie ook meer consistent met Open Zaak en Open Formulieren etc.

File: docs/admin/Metadata/index.rst [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+* Bovenaan zit een zoekveld met en knop **"Zoeken"** waarmee naar *informatiecategorieën* gezocht kan worden.
+* Daaronder zit de mogelijkheid om **eenzelfde actie uit te voeren over meerdere informatiecategorieën**. Op dit moment wordt alleen de actie **"Geselecteerde informatiecategorieën verwijderen"** ondersteund. Merk op dat het mogelijk is om in de lijst één of meerdere *informatiecategorieën* aan te vinken.
+* Onder de (bulk-)actie staat de lijst met *informatiecategorieën*. Door op de kolomtitels te klikken kan de lijst **alfabetisch of chronologisch geordend** worden. 
+* Rechts naast de lijst bestaat de mogelijkheid om deze te **filteren op oorsprong**.
+* Bij een *informatiecategorie* kan op de `naam` geklikt worden om **de details in te zien** en deze eventueel **te wijzigen**.
+* Bij een *informatiecategorie* kan op **"Show logs"** (op één na rechter kolom) geklikt worden om direct de `audittrail`_ in te zien.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:55]: self-reminder: vertaling bijwerken zodra die opgevoerd is

Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:55]:
```suggestion
* Bij een *informatiecategorie* kan op **"Show logs"** (op één na rechter kolom) geklikt worden om direct de :ref:`audit trail <admin_logging_index>` in te zien.
```
-> dit refereert naar het label `admin_logging_index` op een andere plek in de docs, en maakt de linktitel "audit trail"

File: docs/admin/Metadata/index.rst [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+
+Wanneer bij een *informatiecategorie* op  de `naam` wordt geklikt, wordt een scherm geopend met nadere details.
+Hierop zien we:
+
+* **Alle gegevens**. Deze lichten we hieronder toe.
+* Rechtsboven een knop **"Show logs"**. Deze toont de volledige `audittrail`_ van de *informatiecategorie*.
+* Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de Admin-interface zijn uitgevoerd op de *informatiecategorie*.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:62]:
```suggestion
* Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de beheerinterface zijn uitgevoerd op de *informatiecategorie*.
```
Voorkeur voor Nederlands?

File: docs/admin/Metadata/index.rst [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+* Als de *informatiecategorie* tot de landelijke 17 behoort, linksonder een knop om het scherm te **sluiten** en teurg te keren naar de lijst.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:63]:
```suggestion
* Als de *informatiecategorie* tot de landelijke lijst behoort, linksonder een knop om het scherm te **sluiten** en terug te keren naar de lijst.
```
maakt het wat makkelijker als het aantal in de toekomst wijzigt? :)

File: docs/admin/Metadata/index.rst [hunk 20]
```diff
@@ -0,0 +1,115 @@
...
+* Als de *informatiecatgeorie* door de organisatie zelf is toegevoegd, linksonder de mogelijkheid om **wijzigingen op te slaan** (indien van toepassing). Er kan voor gekozen worden om na het opslaan direct een nieuwe *informatiecategorie* aan te maken of om direct de huidige *informatiecategorie* nogmaals te wijzigen.
+* Rechtsonder de mogelijkheid om de *informatiecategorie* te **verwijderen**.
+
+Op een informatiecategorie zijn de volgende gegevens beschikbaar. Op het scherm wordt verplichte velden **dikgedrukt** weergegeven.
+
+* ``Naam``. De naam van de *informatiecategorie*, bijvoorbeeld "convenant".
+* ``Naam meervoud``. De meervoudsvorm van de naam, bijvoorbeeld "convenanten".
+* ``Definitie``. De (mogelijk landelijke) definitie van de *informatiecategorie*.
+* ``UUID``. Een niet-muteerbaar, automatisch toegekend identificatie kenmerk.
+* ``Oorsprong``. Een niet-muteerbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:73]:
```suggestion
Naam
    De naam van de *informatiecategorie*, bijvoorbeeld "convenant".

Naam meervoud
    De meervoudsvorm van de naam, bijvoorbeeld "convenanten".

Definitie
    De (mogelijk landelijke) definitie van de *informatiecategorie*.

UUID
    Een niet-wijzigbaar, automatisch toegekend identificatiekenmerk.

Oorsprong
    Een niet-wijzigbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.
```

* ben niet zeker of deze deifnition list syntax per se beter is, laat ik aan jou om te bepalen
* muteerbaar -> wijzigbaar ivm. toegankelijk taalgebruik waarbij eenvoudigere woorden de voorkeur hebben.

File: docs/admin/Metadata/index.rst [hunk 21]
```diff
@@ -0,0 +1,115 @@
...
+
+Thema's
+--------
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:76]:
```suggestion
Thema's
-------
```

File: docs/admin/Metadata/index.rst [hunk 22]
```diff
@@ -0,0 +1,115 @@
...
+
+Toelichting
+~~~~~~~~~~~~~
+Om de vindbaarheid van openbare documenten te bevorderen ondersteunt de `DiWoo`_-standaard het toekennen van een of meerdere *thema's* aan openbare documenten. Hiervoor is een landelijke waardelijst gedefinieerd: `Thema-indeling voor Officiële Publicaties (TOP-lijst)`_. Op de `Woo-index`_ kan een burger zoeken naar openbare documenten, die aan een bepaald thema gekoppeld zijn.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:80]:
```suggestion
Toelichting
~~~~~~~~~~~

Om de vindbaarheid van openbare documenten te bevorderen ondersteunt de `DiWoo`_-standaard het toekennen van een of meerdere *thema's* aan openbare documenten. Hiervoor is een landelijke waardelijst gedefinieerd: `Thema-indeling voor Officiële Publicaties (TOP-lijst)`_. Op de `Woo-index`_ kan een burger zoeken naar openbare documenten, die aan een bepaald thema gekoppeld zijn.
```

File: docs/admin/Metadata/index.rst [hunk 23]
```diff
@@ -0,0 +1,115 @@
...
+
+De landelijke thema-lijst is ook standaard ingelezen in en wordt ontsloten met het "Woo-publications"-component, zodat deze gebruikt kan worden bij het registreren en vindbaar maken van openbare documenten.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:82]:
```suggestion
De landelijke thema-lijst is ook standaard ingelezen in en wordt ontsloten met het "Woo-publicaties"-component, zodat deze gebruikt kan worden bij het registreren en vindbaar maken van openbare documenten.
```

File: docs/admin/Metadata/index.rst [hunk 24]
```diff
@@ -0,0 +1,115 @@
...
+
+In het "WOO Publications"-component leggen we de *thema's* vast op de *publicatie* (in ontwikkeling). Dezelfde *thema's* gelden voor de daaraan gekoppelde *documenten*.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:84]:
```suggestion
In het "WOO Publicaties"-component leggen we de *thema's* vast op de *publicatie* (in ontwikkeling). Dezelfde *thema's* gelden voor de daaraan gekoppelde *documenten*.
```

File: docs/admin/Metadata/index.rst [hunk 25]
```diff
@@ -0,0 +1,115 @@
...
+
+De `DiWoo`_-standaard ondersteunt alleen het gebruik van de landelijke waardelijst `Thema-indeling voor Officiële Publicaties (TOP-lijst)`_. Ook op de `Woo-index`_ kan alleen op deze *thema's*  gezocht worden naar openbare documenten.
+
+Beheerscherm
+~~~~~~~~~~~~~~
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:89]:
```suggestion
Beheerscherm
~~~~~~~~~~~~
```

File: docs/admin/Metadata/index.rst [hunk 26]
```diff
@@ -0,0 +1,115 @@
...
+
+* **Alle gegevens**. Deze lichten we hieronder toe.
+* Rechtsboven een knop **"Show logs"**. Deze toont de volledige `audittrail`_ van het *thema*.
+* Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de Admin-interface zijn uitgevoerd op het *thema*.
+* Linksonder een knop om het scherm te **sluiten** en teurg te keren naar de lijst.
+* Rechtsonder de mogelijkheid om het *thema* te **verwijderen**.
+
+Op een *thema* zijn de volgende gegevens beschikbaar.
+
+* ``UUID``. Een niet-muteerbaar, automatisch toegekend identificatie kenmerk.
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:112]:
```suggestion
* ``UUID``. Een niet-wijzigbaar, automatisch toegekend identificatie kenmerk.
```

File: docs/admin/Metadata/index.rst [hunk 27]
```diff
@@ -0,0 +1,115 @@
...
+* ``Naam``. De naam van het *thema*, bijvoorbeeld "cultuur en recreatie".
+* ``position``. (In ontwikkeling)
+* ``ref node id``. (In ontwikkeling)
```
Comment (sergei-maertens) [inline docs/admin/Metadata/index.rst:115]: zijn technische velden voor de boomstructuur, heeft de beheerder verder weinig mee van doen. Kan je afdoen als "systeemvelden"

File: docs/admin/Overige/index.rst [hunk 28]
```diff
@@ -0,0 +1,4 @@
+.. admin_Overige_index:
+
+Overige
+========
```
Comment (sergei-maertens) [inline docs/admin/Overige/index.rst:4]:
```suggestion
Overige
=======
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
This is purely stylistic, but we follow the convention of using lower-cased directory and file names - do you mind changing that?

Other than that - thanks a lot for the content, our approach would definitely have been more minimalistic, but the additional context should really help content editors!

File: docs/admin/Accounts/index.rst (hunk 1 above)
Comment (MarcoKlerks) [reply docs/admin/Accounts/index.rst:4]: @sergei-maertens 
Actually, I found in several RST-manuals that the amount of `=` characters should be **at least as many** as the length of the title. :-)

For instance:
https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html#section-structure 
https://sphinx-tutorial.readthedocs.io/step-1/

Comment (sergei-maertens) [reply docs/admin/Accounts/index.rst:4]: yes, that's correct, but in our styleguides we use the same amount :)

**Comment by sergei-maertens** [comment]
> > This is purely stylistic, but we follow the convention of using lower-cased directory and file names - do you mind changing that?
> > Other than that - thanks a lot for the content, our approach would definitely have been more minimalistic, but the additional context should really help content editors!
> 
> @sergei The only way I can figure out how to do this in GitHub is by downloading and deleting everything and then change and re-upload it. I don't see a button to change folder or file names. Am I correct?

I'm afraid so - if it makes things easier, I can fix this up much quicker in my editor

**Comment by MarcoKlerks** [comment]
> > > This is purely stylistic, but we follow the convention of using lower-cased directory and file names - do you mind changing that?
> > > Other than that - thanks a lot for the content, our approach would definitely have been more minimalistic, but the additional context should really help content editors!
> > 
> > 
> > @sergei The only way I can figure out how to do this in GitHub is by downloading and deleting everything and then change and re-upload it. I don't see a button to change folder or file names. Am I correct?
> 
> I'm afraid so - if it makes things easier, I can fix this up much quicker in my editor

Fixed it manually (download, change and re-upload).

I'm still struggling with the ``:ref:`` references to the audit trail page. They still don't work.

**Comment by MarcoKlerks** [comment]
> I'm still struggling with the `:ref:` references to the audit trail page. They still don't work.

@sergei-maertens Please help! I can't get these references to work and I don't know why. They are on publicaties/index.rst and metadata/index.rst (two on each page).

**Comment by sergei-maertens** [comment]
> > I'm still struggling with the `:ref:` references to the audit trail page. They still don't work.
> 
> @sergei-maertens Please help! I can't get these references to work and I don't know why. They are on publicaties/index.rst and metadata/index.rst (two on each page).

I will look into it after lunch!

**Review by sergei-maertens — APPROVED** [review]
I pushed some small tweaks and fixed some markup issues - nothing major.

**Pull request merged**